specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json

```diff
@@ -7839,7 +7839,8 @@
         "properties": {
           "x-ms-client-flatten": true,
           "$ref": "#/definitions/DataWareHouseQueriesProperties",
-          "description": "Resource properties."
+          "description": "Resource properties.",
+          "x-ms-parameter-location": "method"
```

Comment:
why there is a need to add x-ms-parameter-location and its value as method.
What does method denotes.

Reply:
Can you please point me to the change where location schema is added